<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AppearanceSettingsDialog</class>
 <widget class="QDialog" name="AppearanceSettingsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>820</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Appearance Settings</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>34</x>
     <y>410</y>
     <width>730</width>
     <height>36</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Orientation::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::StandardButton::Apply|QDialogButtonBox::StandardButton::Cancel|QDialogButtonBox::StandardButton::Ok|QDialogButtonBox::StandardButton::RestoreDefaults</set>
   </property>
   <property name="centerButtons">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="minSizeLabel">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>30</y>
     <width>220</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>Minimum icon size</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="minSize">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>30</y>
     <width>81</width>
     <height>40</height>
    </rect>
   </property>
   <property name="minimum">
    <number>16</number>
   </property>
   <property name="maximum">
    <number>64</number>
   </property>
   <property name="singleStep">
    <number>8</number>
   </property>
  </widget>
  <widget class="QLabel" name="maxSizeLabel">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>30</y>
     <width>220</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>Maximum icon size</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="maxSize">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>30</y>
     <width>81</width>
     <height>40</height>
    </rect>
   </property>
   <property name="minimum">
    <number>32</number>
   </property>
   <property name="maximum">
    <number>192</number>
   </property>
   <property name="singleStep">
    <number>8</number>
   </property>
  </widget>
  <widget class="QLabel" name="backgroundTransparencyLabel">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>90</y>
     <width>220</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>Panel transparency</string>
   </property>
  </widget>
  <widget class="QLabel" name="backgroundColorLabel">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>150</y>
     <width>220</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>Background color</string>
   </property>
  </widget>
  <widget class="QLabel" name="borderColorLabel">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>150</y>
     <width>220</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>Border color</string>
   </property>
  </widget>
  <widget class="QLabel" name="tooltipFontSizeLabel">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>270</y>
     <width>220</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>Tooltip font size</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="tooltipFontSize">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>270</y>
     <width>81</width>
     <height>40</height>
    </rect>
   </property>
   <property name="minimum">
    <number>8</number>
   </property>
   <property name="maximum">
    <number>28</number>
   </property>
   <property name="singleStep">
    <number>2</number>
   </property>
  </widget>
  <widget class="QLabel" name="spacingFactorLabel">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>90</y>
     <width>220</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>Icon spacing factor</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="backgroundTransparency">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>90</y>
     <width>81</width>
     <height>40</height>
    </rect>
   </property>
   <property name="suffix">
    <string>%</string>
   </property>
   <property name="maximum">
    <number>100</number>
   </property>
   <property name="singleStep">
    <number>5</number>
   </property>
   <property name="value">
    <number>100</number>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="spacingFactor">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>90</y>
     <width>81</width>
     <height>40</height>
    </rect>
   </property>
   <property name="decimals">
    <number>1</number>
   </property>
   <property name="minimum">
    <double>0.100000000000000</double>
   </property>
   <property name="maximum">
    <double>0.900000000000000</double>
   </property>
   <property name="singleStep">
    <double>0.100000000000000</double>
   </property>
   <property name="value">
    <double>0.500000000000000</double>
   </property>
  </widget>
  <widget class="QLabel" name="floatingMarginLabel">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>330</y>
     <width>200</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>Floating margin</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="floatingMargin">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>330</y>
     <width>81</width>
     <height>40</height>
    </rect>
   </property>
   <property name="minimum">
    <number>2</number>
   </property>
   <property name="maximum">
    <number>32</number>
   </property>
   <property name="singleStep">
    <number>2</number>
   </property>
   <property name="value">
    <number>6</number>
   </property>
  </widget>
  <widget class="QLabel" name="activeIndicatorLabel">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>210</y>
     <width>220</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>Indicator color (active)</string>
   </property>
  </widget>
  <widget class="QLabel" name="inactiveIndicatorLabel">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>210</y>
     <width>220</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>Indicator color (inactive)</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="bouncingLauncherIcon">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>330</y>
     <width>300</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>Bouncing launcher icon</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="showTooltip">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>270</y>
     <width>300</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>Show tooltip</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>AppearanceSettingsDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>AppearanceSettingsDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
